model/config.go

```diff
@@ -261,6 +255,9 @@ const (
 
 	// HeaderAuthority is authority HTTP header
 	HeaderAuthority = "authority"
+
+	// NamespaceAll is a designated symbol for listing across all namespaces
+	NamespaceAll = ""
```

Comment:
Typically "" is used for default value - for all a special value like "*" or "ALL" should probably be used.

Reply:
Again, k8s convention.